data/auxiliary/gather/ldap_query/ldap_queries_default.yaml

```diff
@@ -253,6 +268,11 @@ queries:
       - dnsHostName
     references:
       - https://troopers.de/downloads/troopers19/TROOPERS19_AD_Fun_With_LDAP.pdf
+  - action: ENUM_MACHINE_ACCOUNT_QUOTA
+    description: 'Dump the machine account quota for the domain.'
```

Comment:
This somewhat repeats the name of the action in its description. We should really mention that this actually is related to a user. Specifically, its related to how many computer accounts a user is allowed to create in a domain.

Reply:
Fixed this n pushed up changes.